data/hai/hai reveal 1 intro.txt

```diff
@@ -58,12 +58,12 @@ mission "Event: Pug Media Coverage"
 		event "hai secret leaks" 10 15
 		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid, and insists upon the Navy taking greater steps to better explore the galaxy beyond human space to search for possible threats.`
 		conversation
-			`As you leave your ship, you see two captains ranting about a video playing on a datapad. The walkway forces you to pass close to them.`
+			`As you leave your ship, you see a cloud of captains and crew watching a video on a public information display. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. The walkway from your ship passes by the display, but the video finishes before you arrive. Many of the onlookers have left for their business, but two captains remain. They're having a passionate discussion about the video.`
```

Comment:
```suggestion
			`As you leave your ship, you see a crowd of captains and crewmembers watching a video on a public information display, not far from your landing spot. The video jumps between a panel discussion and footage of Admiral Danforth being quizzed by Parliament. It finishes before you arrive, and many of the onlookers have left for their business, but two captains remain; and by the sounds of it, they're having a passionate discussion about the video.`
```
Clouds make me feel fuzzy inside. Crowds make me feel like there's something important going on.

I know that the dialogue has to be as generic as possible, but I really can't place this anywhere in a spaceport. I suppose, in a thousand years' time, there'll be public TVs broadcasting the news everywhere. I don't think my suggestion is the best way to word what I want. 

It might also be more interesting if the player can't make out Admiral Danforth, but then again, it would be harder to draw in the player without Danforth plastered on it at the start.

Reply:
You're not in a spaceport, and it's not a public broadcasting TV. It's a landing mission, and the display is interactive. Airports and train stations have interactive informational displays in some countries. They're quite a lot larger than your phone, and they often have real-time feeds of information relevant to your location.

Reply:
Alright.